Gemfile

```diff
@@ -31,6 +31,9 @@ group :test do
   gem 'mocha', '>= 1.2.1',                                          :require => false
   gem 'coveralls',                                                  :require => false if RUBY_VERSION >= '2.0.0'
   gem 'simplecov-console',                                          :require => false if RUBY_VERSION >= '2.0.0'
+  gem 'github_changelog_generator', '~> 1.13.0',                    :require => false if RUBY_VERSION < '2.2.2'
```

Comment:
If these dependencies get crazy you could just note that the person doing the release needs to have a specific ruby version, such as the newest supported.